templates/around/map.html

```diff
@@ -25,5 +25,8 @@
     {% endfor %}
     ]
   </script>
-<script src={{ url_for('static', filename='js/around-map.js') }}></script>
+  {% block extra_js %}
+  <script src={{ url_for('static', filename='js/around-map.js') }}></script>
+  <script src={{ url_for('static', filename='components/App-around.js') }}></script>
```

Comment:
See [the React files and folders naming conventions](https://gist.github.com/koistya/d7a507438c741ee6adb5)